### PR TITLE
Review for Project 3

### PR DESCRIPTION
**Pull request by chelmerrox**
I have completed this project.

My web page is divided as such:

1. The navbar that includes the login button

2. The main container that has:

   - Article 1
   - Article 2
   - Article 3
   - Article 4
   - Main container bottom links container (it's like a footer for the main container)
   - Read comments and bottom social media links container

Each article has its heading, images, captions etc. All of the 6 containers listed above act as grid items for the main (grid) container. 
So, the main container is divided into a 6 rows x 3 columns grid. It is a separate grid container.

3. The recommended articles container that has:

  - more space and astronomy container and editor's picks container
    a. more space and astronomy container is a 2 rows x 3 columns grid
    b. editor's picks container is a 1 row x 3 columns grid
  - more popular container (the sidebar)

The recommended articles container is divided into a 1 row x 2 columns grid.

4. Footer 1 container. It's divided as such:

  - Logo and go to home page container
  - footer 1 top links container (listed links in News, Opinion etc)
  -  footer 1 bottom links container (the web page ownership information in small texts)

5. Footer 2 container
  - footer 2 text
  - expand container

### CODE REVIEW

**Comment by lekegitrepo** [comment]
#### Hi @chelmerrox :clap: :clap:
#### You have done an excellent job. There are few requirements missing in the project. Check my comment below :point_down: 

**_1. The "Theory of inflation" section is a combination of image and text check or inspects the original to understand the implementation._**

**_2. Elements should be aligned with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction))_**
**_use of float is required, try to add this implementation in the CSS style rules_**

**_3. Use [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the righmost side. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))_**

**_4. "More in Space and Astronomy" and "Most Popular" sections should contain links to other articles_**

**_Implement these required changes and submit them for review._**

**_if you have any doubts or questions contact me on slack._**

### _Reviewed by Temitayo Adeleke._

**Comment by chelmerrox** [comment]
I have done the changes I needed to do for this project according to the previous TSEs' reviews:

1. Removed the image of the theory of inflation container and instead built it using grid. It is a 2 rows x 4 columns grid.

2. Used float and position properties in the expand container in footer 2 container.

3. Replaced div tag with the aside tag for the editors' picks container

4. Changed the tags used in all sub-containers in the recommended articles container to a tags and linking each article headings to their articles.

5. Removed comments in my index.html file.

File: index.html [hunk 1]
```diff
@@ -4,11 +4,1011 @@
...
         <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
         <link href="styles/CSS-Resets.css" type="text/css" rel="stylesheet"/>
-        <link href="styles/CSS-of-webpage.css" type="text/css" rel="stylesheet"/>
+        <link href="styles/style.css" type="text/css" rel="stylesheet"/>
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
         <title>Project 3 - Positioning and Floating Elements</title>
     </head>
 
     <body>
+        <nav>
```
Comment (rammazzoti2000) [inline index.html:13]: The social buttons are overlaying over the navbar when scrolling, this is due to the position property, which usually must be used with z-index property.
Please add `z-index: 2;`  to nav element

File: index.html [hunk 2]
```diff
@@ -4,11 +4,1011 @@
...
+            <!--Navbar left container-->
+            <div class="navbar-left-container">
+                <i class="material-icons">menu</i>
+                <i class="material-icons search-icon">search</i>
+                <a href="https://www.nytimes.com/section/science/space" class="space-and-cosmos-link">SPACE &amp; COSMOS</a>
+            </div>
 
+            <img src="images/nyt_logo.PNG" alt="New York Times logo" width="170" height="30"/>
```
Comment (rammazzoti2000) [inline index.html:21]: The NY Times logo is a [link to the page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)

File: index.html [hunk 3]
```diff
@@ -4,11 +4,1011 @@
...
+                
+                <div class="paragraphs-container">
+                    <p class="main-article-author-name">By Dennis Overbye</p>
+                    <br>
+
+                    <div class="date-and-social-media-container">
+                        <p class="article-date">March 17, 2014</p>
+
+                        <!--The code for the icons is from the source page itself: https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0-->
+                        <div class="top-social-media-container">
```
Comment (rammazzoti2000) [inline index.html:52]: The social buttons are overlaying over the navbar when scrolling, this is due to the position property, which usually must be used with z-index property.
Please add `z-index: 1;` to `top-social-media-container`

File: index.html [hunk 4]
```diff
@@ -4,11 +4,1011 @@
...
+                        <p>Astrophysics detected ripples in the fabric of space-time — so-</p>
+                        <p>called <span class="paragraph-links">gravitational waves</span> — the signature of a universe being</p>
+                        <p>wrenched violently apart when it was roughly a trillionth of a</p>
+                        <p>trillionth of a trillionth of a second old. They are the long-sought</p>
+                        <p>smoking-gun evidence of inflation, proof, Dr. Kovac and his</p>
+                        <p>colleagues say, that Dr. Guth was correct.</p>
+                    </div>
+                    <br>
+                    
+                    <div class="thank-you-container">
```
Comment (rammazzoti2000) [inline index.html:140]: `thank you container` should be aligned along with the main body and not with the page, try to move this container inside the `paragraph-styling` container and find a way to align it properly.

File: index.html [hunk 5]
```diff
@@ -4,11 +4,1011 @@
...
+
+        <div class="recommended-articles-container">
+            <div class="recommended-articles-grid-item-1">
+                <div class="space-and-astronomy-border-top"></div> 
+
+                <h4 class="recommended-articles-headings">More in Space and Astronomy</h4>
+
+                <div class="space-and-astronomy-container">
+                    
+                    <div class="space-and-astronomy-container-grid-item-1">
```
Comment (rammazzoti2000) [inline index.html:696]: The description of each article in `More in Space and Astronomy` should be better aligned, the text is overflowing outside the container. Group the image and description together to limit the width of the description and not overflowing

File: index.html [hunk 6]
```diff
@@ -4,11 +4,1011 @@
...
+                        <img src="images/image_11.PNG" alt="SpaceX"/>
+                        <p class="recommended-articles-author authors-names">Steve Nesius/Reuters</p>
+                        <a href="https://www.nytimes.com/2020/04/17/science/spacex-nasa-crew-dragon.html" class="headings">SpaceX Gets May Date to Launch 2
+                        <span>NASA Astronauts to Space Station </span></a>
+                        <p class="dates">April 17</p>
+                    </div>
+                    
+                    <div class="space-and-astronomy-container-grid-item-2">
```
Comment (rammazzoti2000) [inline index.html:704]: The description of each article in `More in Space and Astronomy` should be better aligned, the text is overflowing outside the container. Group the image and description together to limit the width of the description and not overflowing

File: index.html [hunk 7]
```diff
@@ -4,11 +4,1011 @@
...
+                        <img src="images/image_12.PNG" alt="Big Bang"/>
+                        <p class="recommended-articles-author">Kamioka Observatory, Institute for Cosmic Ray Research</p>
+                        <p class="recommended-articles-author">University of Tokyo</p>
+                        <a href="https://www.nytimes.com/2020/04/15/science/physics-neutrino-antimatter-ichikawa-t2k.html" class="headings">Why the Big Bang Produced
+                        <span>Something Rather Than Nothing</span></a>
+                        <p class="dates">April 27</p>
+                    </div>
+                    
+                    <div class="space-and-astronomy-container-grid-item-3">
```
Comment (rammazzoti2000) [inline index.html:713]: The description of each article in `More in Space and Astronomy` should be better aligned, the text is overflowing outside the container. Group the image and description together to limit the width of the description and not overflowing

File: index.html [hunk 8]
```diff
@@ -4,11 +4,1011 @@
...
+                        <img src="images/image_13.PNG" alt="Black Hole"/>
+                        <p class="recommended-articles-author">L. Calçada/European Southern Observatory</p>
+                        <a href="https://www.nytimes.com/2020/04/16/science/black-hole-sagittarius-a.html" class="headings">Dancing With a Black Hole</a>
+                        <p class="dates">April 17</p>
+                    </div>
+                    
+                    <div class="space-and-astronomy-container-grid-item-4">
```
Comment (rammazzoti2000) [inline index.html:720]: The description of each article in `More in Space and Astronomy` should be better aligned, the text is overflowing outside the container. Group the image and description together to limit the width of the description and not overflowing

File: index.html [hunk 9]
```diff
@@ -4,11 +4,1011 @@
...
+                        <img src="images/image_14.PNG" alt="Apollo 13 Astronauts"/>
+                        <p class="recommended-articles-author authors-names">NASA</p>
+                        <a href="https://www.nytimes.com/2020/04/13/science/apollo-13-anniversary.html" class="headings">Apollo 13's Astronauts Survived
+                        <span>Disaster 50 Years Ago. Could it</span>
+                        <span>Happen Again</span></a>
+                        <p class="dates">April 13</p>
+                    </div>
+                    
+                    <div class="space-and-astronomy-container-grid-item-5">
```
Comment (rammazzoti2000) [inline index.html:729]: The description of each article in `More in Space and Astronomy` should be better aligned, the text is overflowing outside the container. Group the image and description together to limit the width of the description and not overflowing

File: index.html [hunk 10]
```diff
@@ -4,11 +4,1011 @@
...
+                        <img src="images/image_15.PNG" alt="Apollo 13"/>
+                        <p class="recommended-articles-author authors-names">NASA</p>
+                        <a href="https://www.nytimes.com/interactive/2020/04/13/science/apollo-13-moon-photos-as-they-shot-it.html" class="headings">Apollo 13: As They Shot It</a>
+                        <p class="dates">April 14</p>
+                    </div>
+                    
+                    <div class="space-and-astronomy-container-grid-item-6">
```
Comment (rammazzoti2000) [inline index.html:736]: The description of each article in `More in Space and Astronomy` should be better aligned, the text is overflowing outside the container. Group the image and description together to limit the width of the description and not overflowing

File: index.html [hunk 11]
```diff
@@ -4,11 +4,1011 @@
...
+                        <span>System and Now It's Breaking Up</span></a>
+                        <p class="dates">April 17</p>
+                    </div>
+                </div>
+
+                <aside class="editors-picks-container-top">
+                    <h4 class="recommended-articles-headings">Editors' Picks</h4>
+
+                    <div class="editors-picks-container">
+                        <div class="editors-picks-container-grid-item-1">
```
Comment (rammazzoti2000) [inline index.html:749]: The description of each article in `Editor's picks` should be better aligned, the text is overflowing outside the container. Group the image and description together to limit the width of the description and not overflowing

File: index.html [hunk 12]
```diff
@@ -4,11 +4,1011 @@
...
+                            <img src="images/image_17.PNG" alt="Stephen King">
+                            <p class="recommended-articles-author authors-names">Illustration by Jules Julien</p>
+                            <a href="https://www.nytimes.com/interactive/2020/04/27/magazine/stephen-king-interview.html" class="headings">Stephen King Has an Idea for the
+                            <span>Story Joe Biden Could Be Telling</span></a>
+                            <p class="dates">April 28</p>
+                        </div>
+                        
+                        <div class="editors-picks-container-grid-item-2">
```
Comment (rammazzoti2000) [inline index.html:757]: The description of each article in `Editor's picks` should be better aligned, the text is overflowing outside the container. Group the image and description together to limit the width of the description and not overflowing

File: index.html [hunk 13]
```diff
@@ -4,11 +4,1011 @@
...
+                            <img src="images/image_18.PNG" alt="Covid-19 not a Hoax">
+                            <p class="recommended-articles-author authors-names">Illustration by Tomi Um</p>
+                            <a href="https://www.nytimes.com/2020/04/22/magazine/coronavirus-hoax.html" class="headings">How Do I Deal With a Friend Who
+                            <span>Thinks Covid-19 Is a Hoax?</span></a>
+                            <p class="dates">April 22</p>
+                        </div>
+                        
+                        <div class="editors-picks-container-grid-item-3">
```
Comment (rammazzoti2000) [inline index.html:765]: The description of each article in `Editor's picks` should be better aligned, the text is overflowing outside the container. Group the image and description together to limit the width of the description and not overflowing

File: index.html (hunk 1 above)
Comment (chelmerrox) [reply index.html:13]: Thanks! I have resolved this problem.

File: index.html (hunk 3 above)
Comment (chelmerrox) [reply index.html:54]: Thanks! I have resolved this problem.

File: index.html (hunk 4 above)
Comment (chelmerrox) [reply index.html:142]: Thanks! I have resolved this problem.

File: index.html (hunk 9 above)
Comment (chelmerrox) [reply index.html:738]: Thanks! I have resolved this problem.

File: index.html (hunk 11 above)
Comment (chelmerrox) [reply index.html:762]: Thanks! I have resolved this problem.

File: index.html (hunk 13 above)
Comment (chelmerrox) [reply index.html:782]: Thanks! I have resolved this problem.

File: index.html (hunk 2 above)
Comment (chelmerrox) [reply index.html]: Thanks! I have resolved this problem.

**Comment by chelmerrox** [comment]
I have done the following changes according to the previous TSEs' review:

 1. Fixed the overlay of social media buttons on navbar when scrolling.

 2. Linked the NYT Logo to the home page.

 3. Aligned the thank-you-container along with the main body

 4. Fixed the alignment of the title description of each article in recommended articles container by 
   containing each grid item in a div with a class and adding width to it in the style.css file.

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Pull request merged**